packages/flame_network_assets/example/pubspec.yaml

```diff
@@ -0,0 +1,18 @@
+name: example
+description: A flame network assets example.
+publish_to: 'none'
+
+version: 1.0.0+1
+
+environment:
+  sdk: '>=2.18.0 <3.0.0'
+
+dependencies:
+  flame: ^1.6.0
+  flame_network_assets:
+    path: ../
```

Comment:
Is this still needed, can't you just write the version with a carret here and melos will take care of it even if it isn't released?

Reply:
Yeah, definitely, this went in the automatic since most of projects that I work on doesn't have melos :P, changed!!